InfiniteRecharge/src/main/java/frc/robot/subsystems/Intake.java

```diff
@@ -0,0 +1,39 @@
+package frc.robot.subsystems;
+
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+
+
+import frc.robot.RobotMap;
+
+public class Intake extends Subsystem {
+
+
+
+    public double getPosition() {
+        return 0.0; //position
+      
+    }
+
+    public boolean zeroPosition() {
+        return false;
+
+    }
+
+    public boolean extendIntake() {
+        //Motion profile stuff
+        RobotMap.getIntakeActuationSystem().set(1);
```

Comment:
See RobotWrangler in codeskeleton, it has its setSystem(SensoredSystem) method so that it doesn't need to access RobotMap directly. It creates a private instance of its SensoredSystem in the class.